update what is tizen (#1880)
Samsung/tizen-docs: docs/platform/what-is-tizen/overview.md

## Connect everything

As diverse devices are getting smarter and more connected, there is a strong need for a software platform for integrated services with connected products. Tizen is designed for this and has been evolving to connect everything.

> To support different types of devices with a seamless user experience, Tizen provides the features and functions for various kind of devices. Tizen supports both headed and headless products. In the near future, lots of internet connected devices are anticipated to be on the market and IoT services are expected to pervade everyone's daily life. Tizen is moving forward to the future of everything connected.

Author: applied

**Figure: Connecting different devices**

![Connecting different devices](media/about_tizen_1_new.png)

## Openness, optimization, and opportunities

> Tizen is a Linux-based, open source software platform. Anyone can contribute and freely develop the system software and applications, which is beneficial for both device manufacturers and application developers. Furthermore, by supporting web applications with open web technologies and C# applications with .Net technologies, Tizen provides developers flexibility and extensibility. Based on the open ecosystem, the user can expect various experiences from Tizen devices and services. With its openness, Tizen can pursue continuing progress and expand the ecosystem.

Author: applied

> Tizen is built to work on diverse devices.

Author: applied

- [TV device](profiles/tv.md)
- [Tizen custom device](profiles/tizen_custom.md)
- [Wearable device](profiles/wearable.md)
- [Mobile device](profiles/mobile.md)

Tizen devices provide product-optimized functionality and performance.

Tizen allows developers to make various kinds of devices and develop applications for the devices.
Tizen provides a building block pool of components mainly based on the Tizen API sets.

> Tizen custom device can be created by combining API building blocks in various ways as per the requirements. Required packages are installed based on the dependencies related to the specified building blocks.

Author: applied

For more information, please reference [Tizen custom device](profiles/tizen_custom.md).

Devices and features under new categories or devices can be easily added for emerging technologies. Therefore, Tizen is adaptable and customizable for new products. In the foreseeable future, for example, IoT devices and services are expected to be prevalent and Tizen is taking proactive steps toward the opportunities in these new markets.

**Figure: Tizen for the various devices**

![Tizen for various devices](media/about_tizen_2_new1.png)

.
## Convergence platform for the emerging era

Since 2012, Tizen has been expanding as smart gadgets have become generalized and connected. By accommodating various device types, Tizen is gearing up for the new era of convergence. As of now, there is a wide variety of Tizen devices on the market:

> - TV & Audio : Samsung Smart TVs, Samsung Lyfestyle TV, Samsung Smart Monitor, Samsung Sound Bar

Author: applied

> - Display : Samsung Smart Signage, Samsung Flip, Samsung Kiosk

Author: applied

> - Smart appliances: Samsung Family Hub Fridge, Samsung air conditioner, Samsung washing machine

Author: applied

- Wearables: Samsung Galaxy Watch series (Galaxy Watch, Galaxy Watch Active, Galaxy Watch Active2, Galaxy Watch3), Gear series (Gear 2, Gear 2 Neo, Gear S, Gear S2, Gear S3)
- Mobile phones: Samsung Z series (Z1(3G), Z3(3G/LTE), Z2(LTE), Z4)

> - Tablets: DIGMA Plane 8501 3G

Reviewer: ```suggestion
- Tablets: Digma Plane 8501 3G
```

Author: applied

> - Cameras: Samsung NX-series(NX300, NX30, NX2000, NX1, NX500), Gear 360

Reviewer: ```suggestion
- Cameras: Samsung NX series (NX300, NX30, NX2000, NX1, NX500), Gear 360
```

Author: applied

> - OBD : Samsung Connect Auto

Reviewer: ```suggestion
- OBD: Samsung Connect Auto
```

Author: applied

> Samsung is holding the highest market share in the global TV market and its recent models are all based on the Tizen platform. It is expected for the Tizen platform to broaden its ecosystem with increasing market requirements and user demands.

Reviewer: ```suggestion
Samsung holds the highest market share in the global TV market and its recent models are all based on the Tizen platform. It is expected that the Tizen platform to broaden its ecosystem with increasing market requirements and user demands.
```

Author: applied
